.github/workflows/go.yaml

```diff
@@ -34,7 +34,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        go: [ '1.16.x']
+        go: ["1.16.x"]
```

Comment:
Should we update this?

Reply:
I'm happy to add this to my open source todo list. I can land it next week. :)